common/gpu/intel/meteor-lake/default.nix

```diff
@@ -0,0 +1,17 @@
+{pkgs, lib, config, ...}: let inherit(lib) mkDefault mkIf; in {
+  imports = [ ../. ];
+
+	# only needed to support the i915 driver, can be found using `nix-shell -p pciutils --run "lspci -nn" | grep -oP "VGA.*:\K[0-9a-f]{4}"`
+	options.hardware.intelgpu.deviceID = lib.mkOption { description = "Intel GPU to probe"; };
+
+	config = {
+		# i915 is buggy on meteor lake, xe should be the default
+		hardware.intelgpu.driver = mkDefault "xe";
+
+		# xe driver requires newer kernel
+		boot.kernelPackages = mkIf(config.hardware.intelgpu.driver=="xe")( mkDefault pkgs.linuxPackages_latest );
```

Comment:
The chosen formatting here looks a bit uncommon, could reformat the file with nixfmt?

Reply:
I conceptually detest the notion of a code formatter, but you can if you must.

Reply:
Ok. but you also seem to mix spaces and tabs, if you look at the code diff.

Reply:
Maybe later, if I have time for it.